plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/tasks/DownloadSDLTask.kt

```diff
@@ -0,0 +1,45 @@
+package com.expediagroup.graphql.plugin.gradle.tasks
+
+import com.expediagroup.graphql.plugin.downloadSchema
+import kotlinx.coroutines.runBlocking
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.RegularFile
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+
+internal const val DOWNLOAD_SDL_TASK: String = "downloadSDL"
+
+@Suppress("UnstableApiUsage")
+open class DownloadSDLTask : DefaultTask() {
+
+    @Input
+    @Option(option = "endpoint", description = "target SDL endpoint")
+    val endpoint: Property<String> = project.objects.property(String::class.java)
+
+    @Input
+    @Option(option = "outputFileName", description = "target schema file name")
```

Comment:
As I have learned from our own plugins, if the expected file format is different from the `downloadSdl` plugin to `instrospectSchema` plugin, can we have the output argument names be different.

This will make it easier if we happen to want to run both plugins in one command